vaadin-grid-pro-flow-demo/src/main/java/com/vaadin/flow/component/gridpro/vaadincom/GridProView.java

```diff
@@ -120,6 +122,27 @@ private void customRepresentation() {
         addCard("Custom Representation", grid);
     }
 
+    private void customEditorType() {
+        // begin-source-example
+        // source-example-heading: Custom Editor Type
+        GridPro<Person> grid = new GridPro<>();
+        grid.setItems(createItems());
+
+        grid.addColumn(Person::getName).setHeader("NAME");
+
+        /*
+         * Using ComponentRenderer to create a custom representation of the boolean value.
```

Comment:
Remove this comment, as it seems to be a copy-paste from above and is incorrect in this example